src/main/java/edu/kis/powp/jobs2d/command/FactoryCommand.java

```diff
@@ -0,0 +1,24 @@
+package edu.kis.powp.jobs2d.command;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import java.util.ArrayList;
+import java.util.List;
+public class FactoryCommand {
+    public static ComplexCommand drawTrinagle(Job2dDriver driver){
```

Comment:
The name is incorrect. Factory doesn't draw it creates an instance of a triangle command.
The return type could be IDriverCommand.